src/EventStore.Core/ClusterVNode.cs

```diff
@@ -282,12 +282,16 @@ public class ClusterVNode<TStreamId> :
 					$"{nameof(options.Database.InitializationThreads)} must be greater than 0.");
 			}
 
-			if (options.Grpc.KeepAliveTimeout <= 0) {
-				throw new ArgumentOutOfRangeException(nameof(options.Grpc.KeepAliveTimeout));
+			if (options.Grpc.KeepAliveTimeout < 0) {
+				throw new ArgumentOutOfRangeException($"Invalid keepAliveTimeout {options.Grpc.KeepAliveTimeout}. Please provide a positive integer.");
```

Comment:
I suggest using `nameof(setting)` in these messages so that the error doesn't need to be updated if the option's name changes:

```suggestion
				throw new ArgumentOutOfRangeException($"Invalid {nameof(options.Grpc.KeepAliveTimeout)} {options.Grpc.KeepAliveTimeout}. Please provide a positive integer.");
```